<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerEmulatorDeviceDialog</class>
 <widget class="QDialog" name="Mer::Internal::MerEmulatorDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>503</width>
        <height>257</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ConfigName:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="configNameLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="emulatorLabel">
            <property name="text">
             <string>Emulator Vm:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="emulatorComboBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="merVmLabel">
            <property name="text">
             <string>Mer Vm:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="merVmComboBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="userNameLabel">
            <property name="text">
             <string>&amp;Username:</string>
            </property>
            <property name="buddy">
             <cstring>userLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="userLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="sshKeyLabel">
            <property name="text">
             <string>SSH key:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="sshKeyLabelEdit">
            <property name="text">
             <string>none</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="sshPortLabel">
            <property name="text">
             <string>&amp;SSH port:</string>
            </property>
            <property name="buddy">
             <cstring>sshPortSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="sshPortSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>22</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="connectionTimeoutLabel">
            <property name="text">
             <string>Timeout:</string>
            </property>
            <property name="buddy">
             <cstring>timeoutSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="timeoutSpinBox">
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="freePortsLabel">
            <property name="text">
             <string>Free ports:</string>
            </property>
            <property name="buddy">
             <cstring>portsLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="portsLineEdit">
              <property name="toolTip">
               <string>You can enter lists and ranges like this: 1024,1026-1028,1030</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="portsWarningLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="1">
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="standardButtons">
             <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>userLineEdit</zorder>
         <zorder>userNameLabel</zorder>
         <zorder>sshPortLabel</zorder>
         <zorder>sshPortSpinBox</zorder>
         <zorder>connectionTimeoutLabel</zorder>
         <zorder>timeoutSpinBox</zorder>
         <zorder>freePortsLabel</zorder>
         <zorder>merVmLabel</zorder>
         <zorder>sshKeyLabel</zorder>
         <zorder>sshKeyLabelEdit</zorder>
         <zorder>merVmComboBox</zorder>
         <zorder>emulatorComboBox</zorder>
         <zorder>emulatorLabel</zorder>
         <zorder>buttonBox</zorder>
         <zorder>configNameLineEdit</zorder>
         <zorder>label</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Mer::Internal::MerEmulatorDeviceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Mer::Internal::MerEmulatorDeviceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
